css/index.css

```diff
@@ -32,32 +33,77 @@ font-size: 24px;
 color: #525252;
 }
 
-ul {
+/*ul {
   padding-left: 0;
   list-style-position: inside;
-}
+}*/
 
 #About {
+  padding: 10px 0;
+  width: 500px;
   text-align: center;
-  word-wrap: normal;
-  padding: 5px 0px;
-  margin: 2px;
+  margin: auto;
+  font-size: 18px;
 }
 
 #Experience {
   text-align: center;
-  list-style-type: square;
-  padding: 5px 0px;
-  margin: 2px;
+  padding: 10px 0;
+  margin: auto;
+  width: 500px;
 }
 
 #Connect {
   text-align: center;
-  padding: 5px 0px;
-  margin: 2px;
+  padding: 10px 0;
+  width: 500px;
+  margin: auto;
 }
 
+/* Changes from last week - removed since I no longer have the list
+#Experience li {
+  list-style-type: square;
+  padding: 5px;
+  background-color: #dab1da;
+}
+*/
 
+.topnav {
+  padding: 10px;
+  position: sticky;
+  position: -webkit-sticky;
+  top: 0;
+  letter-spacing: .5px;
+  text-transform: uppercase;
+  text-align: center;
+}
 
+#Skills {
+  padding: 10px 0;
+  text-align: center;
+}
 
+#Projects {
+  padding: 10px 0;
+  text-align: center;
+}
+
+.allexperience {
+  display: flex;
+}
+
+.eachexperience {
+  text-align: center;
+  margin: 0 10px;
+  font-size: 16px;
+}
+
+.allconnect {
+  display: flex;
```

Comment:
to center the items, you need to center the container flex styling, something like 
`justify-content: center;` worked for me when I tried it on your current file